modules/storages/spec/features/admin_storages_spec.rb

```diff
@@ -156,9 +162,16 @@
     # List of storages
     page.find("ul.op-breadcrumb li", text: "File storages").click
 
-    # Go to Other NC again
-    page.find("a", text: 'Other NC').click
-    expect(page).to have_current_path admin_settings_storage_path(created_storage)
+    # Check created OAuth2 Application
+    page.find('[data-qa-selector="op-menu--item-action"]', text: 'Authentication').click
+    page.find('[data-qa-selector="op-menu--item-action"]', text: 'OAuth applications').click
+    expect(page).to have_text("Other NC (#{storage_type})")
+    expect(page).not_to have_selector('a', text: 'Delete')
+
+    # Editing oauth app opens storage edit controller
+    page.find('a', text: 'Edit').click
+    expect(page).to have_title("Edit: Other NC")
+    page.find('button[type=submit]', text: "Save").click
```

Comment:
@apfohl Apart from the "Edit" button, covering the link on the name is missing here, I believe. It should lead to the show view of that storage.

Reply:
I just realized that behaviour and I find it wrong. Clicking on the oauth app link should open the oauth app, not the storage view. And the delete button must be hidden there, and the edit button redirected. but it is inconsistent, that the storage is opened here.